staging/src/k8s.io/apiserver/pkg/storage/selection_predicate.go

```diff
@@ -71,13 +71,18 @@ func (f AttrFunc) WithFieldMutation(fieldMutator FieldMutationFunc) AttrFunc {
 
 // SelectionPredicate is used to represent the way to select objects from api storage.
 type SelectionPredicate struct {
-	Label               labels.Selector
-	Field               fields.Selector
-	GetAttrs            AttrFunc
-	IndexLabels         []string
-	IndexFields         []string
-	Limit               int64
-	Continue            string
+	Label       labels.Selector
+	Field       fields.Selector
+	GetAttrs    AttrFunc
+	IndexLabels []string
+	IndexFields []string
+	Limit       int64
+	Continue    string
+	// AllowWatchBookmarks when set indicates that the end-client
+	// wants to receive optional bookmark events.
+	//
+	// As of today, this will be set when the 'allowWatchBookmarks'
+	// query parameter is set.
```

Comment:
I'm still against this comment:
- if you know what "watch bookmarks" are, then the name is self-explonatory, if you don't know what watch bookmarks are, this comment doesn't help anyway
- this is fairly low in the stack so talking about "end-client" doesn't make sense, it could be set by other layers of the stack and caught or sth...
- at this level, we shouldn't be talking abotu "query parameters" as these aren't really relevant here

So I still don't think this comment provides value...

Reply:
ok, removed, PTAL.